stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/WebSocketUtil.java

```diff
@@ -195,4 +193,13 @@ public static void readFirstFrame(WebSocketConnection webSocketConnection, BStru
         webSocketConnection.readNextFrame();
         webSocketConnector.setBooleanField(WebSocketConstants.CONNECTOR_IS_READY_INDEX, 1);
     }
+
+    /**
+     * Closes the connection with the unexpected failure status code.
+     * @param webSocketConnection the websocket connection to be closed.
+     */
+    static void closeDuringUnexpectedCondition(WebSocketConnection webSocketConnection) {
+        webSocketConnection.terminateConnection(1011, "Unexpected failure");
```

Comment:
Since we are sending a standard error shouldn't we say "Unexpected condition" instead of "failure"?